src/pages/index.js

```diff
@@ -37,43 +38,43 @@ const IndexPage = ({ data }) => (
       <Icon
         icon={ICONS.BRIEFCASE}
         size={100}
-        color={props => props.theme.primary.dark}
+        color={({ theme }) => theme.primary.dark}
```

Comment:
Cool 😎 didn't know you could do this?

Is this from the styled-components documentation?

Reply:
No. Object destructuring:
https://javascript.info/destructuring-assignment#object-destructuring